Unity-Technologies/ScriptableRenderPipeline
# 2d/light-fixes

### Purpose of this PR
Fix to make the conditional checks for lights work more robustly and with the updated Universal Render Pipeline Asset

---
### Release Notes
Contains some code improvements and required updates (because of changes to Universal Render Pipeline Asset)

---
### Testing status
**Manual Tests**: What did you do?
- [X ] Opened test project + Run graphic tests locally
- [X ] C# and shader warnings (supress shader cache to see them)

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)
No new tests. Just fixes.

Alternative, launch Yamato (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/2d%252Flight-fixes/.yamato%252Fupm-ci-universal.yml%2523All_Universal/404098/job/dependencies

---
### Overall Product Risks
**Technical Risk**: Low. A few fairly simple bug fixes.
**Halo Effect**: Low. A few fairly simple bug fixes.
